### PR TITLE
Add in-process bundling. Closes #187

### LINKED ISSUE #187: Run the `watch` task in process
This way it's always keeping bundles up to date in dev, and no need to keep a second terminal open

### PR DESCRIPTION
**Pull request by juliangruber**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ menu[menu.length - 1].submenu.push({
   }
 })
 
-app.on('ready', () => {
+const onReady = () => {
```
Comment (yoshuawuyts) [inline index.js]: Minor nit: could we make this a normal function if possible? I'm cool if you like arrow functions but I always find these `var foo = function() {}` style functions super hard to read

**Comment by yoshuawuyts** [comment]
Code looks good to me. I think once the conflict is resolved we can def merge it :sparkles:.

I was thinking tho: would it perhaps make sense to start moving onto `bankai`? - right now this is turning out to be a reasonable amount of project specific code and moving it out of the project might help keep our codebase clean. It would require a few tweaks to `bankai` tho, (e.g. https://github.com/yoshuawuyts/bankai/issues/123) but reckon it might be worth it. Thoughts?

File: index.js (hunk 1 above)
Comment (juliangruber) [inline index.js]: i'm cool with that too, will change

**Comment by juliangruber** [comment]
I agree that it would make sense to move to bankai! After public launch maybe?

**Comment by yoshuawuyts** [comment]
woot, yeah sounds good :sparkles:

**Pull request merged**